target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -306,6 +306,10 @@ rp-n53)
 rt-ac51u)
 	set_wifi_led "$boardname:blue:wifi"
 	;;
+rt-ac54u)
+	ucidef_set_led_netdev "wifi2g" "Wifi 2.4G" "$boardname:blue:wifi" "wlan1"
+    set_usb_led "$boardname:blue:usb"
```

Comment:
I don't think you need this line, since you are already setting up the usb led via the device-tree.